cms/templates/product_page.html

```diff
@@ -121,6 +113,10 @@ <h1>{{ page.title }}</h1>
               <h2>About this course</h2>
               <p>{{ page.about |richtext }}</p>
             </div>
+            {% if page.prerequisites %}
+              <h2>Prerequisites</h2>
+              <p>{{ page.prerequisites |richtext }}</p>
```

Comment:
We don't need to use an external paragraph element, the `richtext` tag automatically adds the external `<p>` tag for the whole content.
```suggestion
              {{ page.prerequisites |richtext }}
```